### DISCUSSION TITLE
Unhealable widget

### DISCUSSION THREAD

**Discussion by spamwax** [rgd87/Aptechka#68] [Q&A]
In 9.0.24 notes I saw a new widget for unhealable, I cannot find it in the options/gui.
Is this something that's not customizable?

**Comment by d87** [comment]
Yeah it's just a texture stretched over the whole frame.
I was going to make a different 5th debuff highlight specifically for situations when someone can't be targeted/healed, but left it as is for now

**Reply by spamwax** [reply]
so I use full black color for frame and check "separate background color" with a red color assigned to it.
Seems the black color completely overrides the unhealable texture! is it possible to somehow show the texture or promote its strata so it's visible for when color is full black?

**Comment by d87** [comment]
it's not covering it, it's just (semi-transparent) black on black. Idk, i could try making it customizable to make it gray, but there's a high chance it'll look like crap then, and it's not a trivial change too. I'll check, but maybe just don't use it for now, it's pretty hard to make this overlay thing visually work even under normal conditions.
Btw do you pvp in shadowlands? Recently I added this CC list feature intended for arenas, which i'm not sure about. It's using text array to show a list of generalized CC effects on frames and their dispel type

**Reply by spamwax** [reply]
don't worry about it then. is it possible to assign unhealable to one of the 4 already debuff highlights through AptechkaUserConfig?

I've been trying to force myself to play pvp but haven't yet, hopefully this week after reset. I'll update if I see any issues

**Comment by d87** [comment]
You can add it to DebuffAlert1-4 in status list

**Reply by spamwax** [reply]
what I meant is that unhealable status can be shown as a, say, pixel glow. if I assign it to DebuffAlerts then a spell/debuff on player will show as the unhealable texture. what I want is when Aptechka detects unhealable on a player then it can be shown as a pixel glow or something.

By the way, this [WeakAura](https://wago.io/NathriaGlows) seems to have all correct spells/debuffs ids for Castle Nathria. maybe it can be used to update the builtin list that you already have for some of them

**Reply by d87** [reply]
There's no actual unhealable status, i'm pretty sure it's impossible to detect. So it's completely tied to known debuffs that cause it and even then this widget is not used anywhere in default config yet.

Oof, i missed a lot
